api/include/mfxstructures.h

```diff
@@ -44,8 +45,9 @@ typedef struct {
         };
     };
 } mfxFrameId;
+MFX_PACK_END()
 
-#pragma pack(push, 4)
+MFX_PACK_BEGIN_USUAL_STRUCT()
```

Comment:
`mfxU64 BufferSize;`

I suggest to have explicit comment if we need to preserve old alignment.

Reply:
ok